src/systemjs.config.js

```diff
@@ -30,7 +30,6 @@
     // packages tells the System loader how to load when no filename and/or no extension
     packages: {
       app: {
-        main: './main.js',
```

Comment:
what do we gain by removing this? I mean, if we are not importing "app" anymore, shouldn't we remove the entire app node?